pluto/src/vis/schematic/Forms.tsx

```diff
@@ -487,9 +458,7 @@ const ValueTelemForm = ({ path }: { path: string }): ReactElement => {
   };
 
   const c = Channel.useName(source.channel as number);
-  useEffect(() => {
-    onChange({ ...value, tooltip: [c] });
-  }, [c]);
+  useEffect(() => onChange({ ...value, tooltip: [c] }), [c]);
 
   return (
     <FormWrapper direction="x" align="stretch">
```

Comment:
Having all four of these items in a row causes the fields to be compressed too far when the window is small. Can you put the options on two rows?

Reply:
Yeah, I moved it to two lines.